### PR TITLE
Add info about Preventing Escalations to Respectful Review Comments

### PR DESCRIPTION
**Pull request by llaakkkk**
At FOSSHackathons the diversity group have discussed topics which we can cover in the documentation about Code Review Process.

### CODE REVIEW

File: contributing/community/review-comments.rst [hunk 1]
```diff
@@ -149,6 +149,17 @@ you don't have to use "Please" all the time. But it wouldn't hurt.
 It may not seem like much, but saying "Thank you" does make others feel
 more welcome.
 
+
+Preventing Escalations
+------------------
```
Comment (wouterj) [inline contributing/community/review-comments.rst]: The title underline should be exactly the same length as the title itself to make restructured text happy

**Review by wouterj — APPROVED** [review]
Just one minor syntax comment, apart from that its perfect (do you want us to wait for feedback of other diversity members?)

File: contributing/community/review-comments.rst [hunk 2]
```diff
@@ -149,6 +149,17 @@ you don't have to use "Please" all the time. But it wouldn't hurt.
...
+
+Sometimes when people receive feedback they may get defensive.
+In that case, it is a better to try to approach the discussion in
```
Comment (greg0ire) [inline contributing/community/review-comments.rst]:
```suggestion
In that case, it is better to try to approach the discussion in
```

**Comment by greg0ire** [comment]
Please associate your email address with your Github account, or change the email in your commits to an address already associated with it, that way you will get credit for the commits :)

**Review by javiereguiluz — APPROVED** [review]
@llaakkkk very nice! Thanks for your contribution.

**Comment by llaakkkk** [comment]
> Just one minor syntax comment, apart from that its perfect (do you want us to wait for feedback of other diversity members?)

@wouterj no, the text is already approved by the diversity team

**Comment by llaakkkk** [comment]
> Please associate your email address with your Github account, or change the email in your commits to an address already associated with it, that way you will get credit for the commits :)

fixed it, thanks)

**Comment by javiereguiluz** [comment]
Thank you @llaakkkk! We merged this in 3.4 brancha and we'll merge into the other branches automatically. GitHub shows this pull request closed instead of merged because of this branch change, but it's merged!

**Comment by greg0ire** [comment]
@llaakkkk it does not look like it worked. The email you would have to [add as a secondary address to your Github account](https://help.github.com/en/articles/adding-an-email-address-to-your-github-account) is the `lmc.eu` one.

**Comment by llaakkkk** [comment]
> @llaakkkk it does not look like it worked. The email you would have to [add as a secondary address to your Github account](https://help.github.com/en/articles/adding-an-email-address-to-your-github-account) is the `lmc.eu` one. When it works, that
> octocat will become your user pic:
> 
> d7d68a77-bc82-47ae-b506-b9b807743544

@greg0ire I understood the reason, I am going to find how I can fix it. Because lmc.eu is my work email

**Comment by greg0ire** [comment]
There will be no way for you to amend those commits so that you use the new address now that they have been merged, so the only way to fix this is to add your work email as a secondary address to your GH account. Personally, I did it, it's not an issue to me since I enabled 2FA.

**Comment by llaakkkk** [comment]
> I think there will be no way for you to amend those commits so that you use the new address now that they have been merged, so the only way to fix this is to add your work email as a secondary address to your GH account. Personally, I did it, it's not an issue to me since I enabled 2FA.

@greg0ire did it. Thank you for help!

**Pull request closed without merging**